jackal_base/scripts/calibrate_compass

```diff
@@ -55,12 +56,13 @@ fi
 
 if [[ -r $CAL_FILE ]]; then
   echo "Calibration generated in $CAL_FILE."
-  read -r -p "Copy calibration to /etc/ros/${ROS_DISTRO}? [Y/n] " response
+  read -r -p "Copy calibration to ${CAL_FINAL_PATH}? [Y/n] " response
   response=${response,,}    # tolower
   if [[ $response =~ ^(no|n)$ ]]; then
```

Comment:
It's not great, but nothing in bash is. The right answer is probably to rewrite this in Python, and maybe move it to a dedicated magnetometer_calibration package, or something along those lines.